lib/committee/response_validator.rb

```diff
@@ -1,16 +1,18 @@
 module Committee
   class ResponseValidator
-    def initialize(link)
+    def initialize(link, validate_errors = false)
```

Comment:
Oof, yeah, 1.9 is a huge pain. Can we change this though to use the `options = {}` pattern so that we can move it over to an optional argument easily once we drop 1.9 support? Like this:

``` ruby
    def initialize(link, options={})
      ...
      @validate_errors = options[:validate_errors]
    end
```